examples/networks.go

```diff
@@ -5,3 +5,6 @@ const MainnetGateway = "https://gateway.elrond.com"
 
 // TestnetGateway represents the testnet gateway address
 const TestnetGateway = "https://testnet-gateway.elrond.com"
+
+// LocalTestnetGateway represents a local testnet proxy address
+const LocalTestnetGateway = "http://localhost:7950"
```

Comment:
this is too custom. Please remove this and instead use the TestnetGateway as the default one

Reply:
removed, there is the option to give custom url for the proxy flag which should cover this as well.